### PR TITLE
Add github actions to test + publish Lokomotive Web UI

### PR DESCRIPTION
**Pull request by joaquimrocha**

### CODE REVIEW

**Comment by meyskens** [comment]
Hi @joaquimrocha could it be possible to add Docker buildx here so we can publish both an x86 and ARM image? Currently the web-ui part of Lokomotive breaks on ARM64

**Comment by joaquimrocha** [comment]
I don't have more bandwidth for that this week, but we can merge this and add the ARM support later.

File: .github/workflows/image-publish.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      tags:
+        description: 'Image tags (comma separated)'
+        required: true
+        default: 'latest'
+jobs:
+  push_to_registry:
+    name: Test building container image
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get tag
```
Comment (illume) [inline .github/workflows/image-publish.yml:14]: Seems this grabs the most recent tag. 

"`name: Get most recent tag`" ?

**Review by illume — APPROVED** [review]
:+1:

**Pull request merged**